e2e/cypress/integration/emoji/custom_emoji_1_spec.js

```diff
@@ -106,21 +106,20 @@ describe('Custom emojis', () => {
         // # Open custom emoji
         cy.uiOpenCustomEmoji();
 
-        // # Click on add new emoji
-        cy.findByText('Add Custom Emoji').should('be.visible').click();
+        // # Click on add custom emoji
+        cy.findByRole('button', {name: 'Add Custom Emoji'}).should('be.visible').click();
 
         // # Type emoji name
         cy.get('#name').type(customEmojiWithColons);
 
         // # Select emoji image
         cy.get('input#select-emoji').attachFile(largeEmojiFile);
 
-        // # Click on Save
-        cy.get('.backstage-form__footer').findByText('Save').click().wait(TIMEOUTS.FIVE_SEC);
-        cy.findByText('Add Custom Emoji').should('be.visible');
+        // # Save custom emoji
+        saveCustomEmoji(testTeam.name);
 
         // # Go back to home channel
-        cy.findByText('Back to Mattermost').should('exist').and('be.visible').click().wait(TIMEOUTS.FIVE_SEC);
```

Comment:
- `should('exist')` is redundant
- `and('be.visible')` is good to have but decided to remove. Note that it should be fine since it's not preceded by a page load or newly rendered component (ex. showing up a modal), other check have been made to the page (https://github.com/mattermost/mattermost-webapp/pull/9343/files#diff-8e2b0bfb3d487ccdaa95dd2af0aa29960ce8013d8df54ef4fa40ea2e72e554dcR230-R232)